### DISCUSSION TITLE
ImageRepository authentication with AWS Elastic Container Registry Not working

### DISCUSSION THREAD

**Discussion by kapilsingh421** [fluxcd/flux2#1866] [Q&A] [answered]
ImageRepository authentication with AWS Elastic Container Registry Not working on ARM64 graviton node
After debugging I found that the image used in the init container to get cred credentials is not supporting Arm64 instances.

image name:-bitnami/kubectl

**Comment by kingdonb** [comment]
Looks like there is an open issue for this at Bitnami:

* https://github.com/bitnami/bitnami-docker-kubectl/issues/22

Thanks for reporting this!

**Reply by kingdonb** [reply]
(Should we convert this to an issue?)

**Reply by stefanprodan** [reply]
Fix in https://github.com/fluxcd/flux2/pull/1877

**Comment by stefanprodan** [comment, marked as answer]
We've update the manifests in flux2 v0.18.0 with an multi-arch image for kubectl.
